internal/cluster/validations/validations.go

```diff
@@ -931,3 +931,13 @@ func ValidateIgnitionImageSize(config string) error {
 
 	return nil
 }
+
+func ValidateArchitectureWithPlatform(architecture *string, platform *models.Platform) error {
+	if swag.StringValue(architecture) == common.ARM64CPUArchitecture {
```

Comment:
would we want to do the same for ``s390x`` and ``ppc64le``? I guess they are out of question for Nutanix as well
or rather check positively that the architecture is x86

either approaches (positive & negative checks) have pros & cons, so I don't have a strong preference for this or that

Reply:
We don't provide these payloads in AI, do we?

I think it would be safer to limit this to x86_64, let me clarify that with Nutanix

Reply:
> We don't provide these payloads in AI, do we?

I think it's quite new that we do. Maybe 1 week ago it were all added, so I expect to see those alongside arm and x86

Reply:
Clarifying on the best way to approach this with @gamli75

Reply:
Reworked it to deny any non-x86_64 architecture, PTAL